lib/cdo/poste.rb

```diff
@@ -217,7 +217,7 @@ def deliver!(mail)
       content_type = mail.header['Content-Type'].to_s
       raise ArgumentError, "Unsupported message type: #{content_type}" unless content_type =~ /^text\/html;/ && content_type =~ /charset=UTF-8/
       sender = mail.from.first
-      raise ArgumentError, "Unsupported sender: #{sender}" unless sender == 'noreply@code.org'
+      raise ArgumentError, "Unsupported sender: #{sender}" unless sender == 'noreply@code.org' || sender == 'pd@code.org'
```

Comment:
Please make this an ALLOWED_SENDERS class constant that stores a set of allowed senders.

ALLOWED_SENDERS = Set.new ['pd@coder.og', 'noreply@code.org']

Then you just say:
ALLOWED_SENDERS.include?(sender)